Integrations/src/main/java/io/deephaven/integrations/learn/Computer.java

```diff
@@ -0,0 +1,86 @@
+package io.deephaven.integrations.learn;
+
+import io.deephaven.base.verify.Require;
+import io.deephaven.db.tables.Table;
+import io.deephaven.db.v2.sources.ColumnSource;
+import org.jpy.PyObject;
+
+/**
+ * Computer adds new updated/modified row indices to the index set, to be used in the deferred calculation.
+ */
+public class Computer {
+
+    private final PyObject modelFunc;
+    private final int batchSize;
+    private final Input[] inputs;
+    private final ColumnSource<?>[][] colSet;
+    private Future current;
+    private int offset;
+
+    /**
+     * Creates a new Computer.
+     *
+     * @param modelFunc     python function to call on the given inputs from a table.
+     * @param inputs        inputs to the model function.
+     * @param batchSize     maximum number of rows for each deferred computation.
+     */
+    public Computer(Table table, PyObject modelFunc, Input[] inputs, int batchSize) {
+
+        Require.neqNull(table, "table");
+        Require.neqNull(modelFunc, "modelFunc");
+        Require.neqNull(inputs, "inputs");
```

Comment:
Need to check every element of `inputs` to ensure that all slots are non-null.  I think there is a require for this.